src/com/facebook/buck/cli/TestCommand.java

```diff
@@ -252,7 +268,7 @@ private TestRunningOptions getTestRunningOptions(CommandRunnerParams params) {
             .setShufflingTests(isShufflingTests)
             .setPathToXmlTestOutput(Optional.ofNullable(pathToXmlTestOutput))
             .setPathToJavaAgent(Optional.ofNullable(pathToJavaAgent))
-            .setCoverageReportFormat(coverageReportFormat)
+            .setCoverageReportFormats(EnumSet.copyOf(Arrays.asList(coverageReportFormats)))
```

Comment:
this will blow up if coverageReportFormats is an empty array. You can refer to https://stackoverflow.com/questions/22886287/enumset-from-array-shortest-variant for examples of how to avoid it.

Reply:
Incidentally the way code written right now array will never be empty, but extra safety measure will never hurt, updated the code according to your suggestion.